.ado/jobs/sample-apps.yml

```diff
@@ -64,19 +64,46 @@ jobs:
           verbosityRestore: Detailed # Options: quiet, normal, detailed
         condition: succeeded()
 
+      - task: PowerShell@2
```

Comment:
Sorry, this is the 3rd... 
would you mind taking this opportunity to factor out the shared steps into a seperate `templates\run-windows-with-certificates.yml`

Reply:
Completed #8015

Reply:
Will backport soon.